great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -5424,10 +5426,19 @@ def _load_fluent_config(self, config_provider: _ConfigurationProvider) -> GxConf
         )
         return GxConfig(fluent_datasources={})
 
-    def _attach_fluent_config_datasources(self, config: GxConfig):
+    def _attach_fluent_config_datasources_and_build_data_connectors(
```

Comment:
✔️

Reply:
This looks a little too much bundled, but I do not have a better suggestion for now.

Reply:
I agree, it feels bad.

Reply:
🙇